### PR TITLE
Add underground line types

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/dto/catalog/LineType.java [hunk 1]
```diff
@@ -61,7 +62,8 @@ public class LineType {
     @Schema(description = "Linear capacity")
     private Double linearCapacity;
 
+    @JsonIgnore
     public LineTypeEntity toEntity() {
-        return new LineTypeEntity(this);
+        throw new UnsupportedOperationException("TODO");
```
Comment (EstherDarkish) [inline src/main/java/org/gridsuite/modification/server/dto/catalog/LineType.java:67]: This is preparatory work for a later development, right ?

**Review by EstherDarkish — APPROVED** [review]

File: src/main/java/org/gridsuite/modification/server/dto/catalog/AerialLineType.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@Schema(description = "Aerial Line Type")
+public class AerialLineType extends LineType {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/modification/server/dto/catalog/AerialLineType.java:25]: Perhaps add `Infos` suffixe to have the same naming convention anywhere

File: src/main/java/org/gridsuite/modification/server/NetworkModificationController.java [hunk 3]
```diff
@@ -178,14 +177,8 @@ public ResponseEntity<Void> stopBuild(@Parameter(description = "Build receiver")
...
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The line types catalog is returned")})
-    public ResponseEntity<List<LineType>> getLineTypesCatalog(@Parameter(description = "type") @RequestParam(name = "category", required = false) LineTypeCategory category) {
-        List<LineType> res;
-        if (category != null) {
-            res = lineTypesCatalogService.getLineTypesCatalog(category);
-        } else {
-            res = lineTypesCatalogService.getAllLineTypesCatalog();
-        }
-        return ResponseEntity.ok().body(res);
+    public ResponseEntity<List<LineType>> getLineTypesCatalog() {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/modification/server/NetworkModificationController.java:180]: `getLinesTypes` ?
We return a list not a catalog DTO

File: src/main/java/org/gridsuite/modification/server/dto/catalog/AerialLineType.java (hunk 2 above)
Comment (sBouzols) [reply src/main/java/org/gridsuite/modification/server/dto/catalog/AerialLineType.java]: ok renamed

File: src/main/java/org/gridsuite/modification/server/NetworkModificationController.java (hunk 3 above)
Comment (sBouzols) [reply src/main/java/org/gridsuite/modification/server/NetworkModificationController.java]: renamed

**Pull request merged**